index.js

```diff
@@ -37,9 +37,9 @@
       // For now work around this bug in PhantomJS
       // https://github.com/ariya/phantomjs/issues/11722
```

Comment:
this issue now appears to be fixed / closed in phantom. I think the better approach is to revert these temporary changes https://github.com/stephenhandley/type-of-is/commit/a22f0fca9ae1edf5c15baa7fe85ea3b3aba0c768